<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QPushButton" name="go_pushButton">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>400</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>429</y>
      <width>800</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/Cqu_A_1.jpg);</string>
    </property>
    <widget class="QLabel" name="p1_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="LinearGradientPattern">
          <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
           <gradientstop position="0.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.495000000000000">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </gradientstop>
           <gradientstop position="0.505000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
           <gradientstop position="1.000000000000000">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </gradientstop>
          </gradient>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>p1_1</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>280</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_2</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>200</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_3</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_4</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>280</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_5</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_6</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_7</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_8</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_9</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_10</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_11</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_12</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_13">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_13</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_14</string>
     </property>
    </widget>
    <widget class="QLabel" name="p1_15">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>p1_15</string>
     </property>
    </widget>
    <zorder>p1_2</zorder>
    <zorder>p1_3</zorder>
    <zorder>p1_4</zorder>
    <zorder>p1_5</zorder>
    <zorder>p1_6</zorder>
    <zorder>p1_7</zorder>
    <zorder>p1_8</zorder>
    <zorder>p1_9</zorder>
    <zorder>p1_10</zorder>
    <zorder>p1_11</zorder>
    <zorder>p1_12</zorder>
    <zorder>p1_13</zorder>
    <zorder>p1_14</zorder>
    <zorder>p1_15</zorder>
    <zorder>p1_1</zorder>
   </widget>
   <widget class="QComboBox" name="to_comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>p1_1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p1_15</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controller">
     <item>
      <widget class="QLabel" name="from_label">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="from_lineedit"/>
     </item>
     <item>
      <widget class="QLabel" name="to_label">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="to_lineedit"/>
     </item>
    </layout>
   </widget>
   <zorder>widget</zorder>
   <zorder>go_pushButton</zorder>
   <zorder>textEdit</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>to_comboBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
